core/commands/multibase.go

```diff
@@ -0,0 +1,118 @@
+package commands
+
+import (
+	"bytes"
+	"fmt"
+	"io/ioutil"
+	"strings"
+
+	cmds "github.com/ipfs/go-ipfs-cmds"
+	"github.com/ipfs/go-ipfs/core/commands/cmdenv"
+	mbase "github.com/multiformats/go-multibase"
+)
+
+var MbaseCmd = &cmds.Command{
+	Helptext: cmds.HelpText{
+		Tagline: "Encode and decode files or stdin with multibase format",
+	},
+	Subcommands: map[string]*cmds.Command{
+		"encode": mbaseEncodeCmd,
+		"decode": mbaseDecodeCmd,
+		"list":   basesCmd,
+	},
+	Extra: CreateCmdExtras(SetDoesNotUseRepo(true)),
```

Comment:
Going to merge for now as this seems like it wouldn't hurt and matches what `ipfs cid` does.

However, I don't think this really works properly without applying the flag to each subcommand.

@lidel let's revisit during the RC process what we want to do here and figure out if `ipfs cid` has similar problems.

Reply:
Continued in  https://github.com/ipfs/go-ipfs/issues/8375#issuecomment-1012182623